media/server/ipc/source/SessionManagementServer.cpp

```diff
@@ -82,6 +88,56 @@ bool SessionManagementServer::initialize(const std::string &socketName, unsigned
     {
         RIALTO_SERVER_LOG_SYS_WARN(errno, "Failed to change the permissions on the IPC socket");
     }
+
+    const uid_t noOwnerChange = -1; // -1 means chown() won't change the owner
```

Comment:
constexpr, according to our naming policy, const values should start with k, in this case kNoOwnerPolicy